packages/ag-charts-enterprise/src/features/color-picker/colorPickerStyles.css

```diff
@@ -24,6 +16,12 @@
     --checker: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="4" height="4"><rect x="0" y="0" width="4" height="4" fill="%23fff"/><path d="M0 0H2V4H4V2H0Z" fill="%23b2b2b2"/></svg>');
 }
 
+.ag-charts-color-picker__content {
+    display: flex;
+    flex-direction: column;
+    color: var(--ag-charts-toolbar-foreground-color);
```

Comment:
This is already in the popover styles, I believe

```suggestion
```

Reply:
Woops, good spot, thanks!